### DISCUSSION TITLE
Missing in maven central

### DISCUSSION THREAD

**Discussion by bringmethebacon** [GSI-CS-CO/chart-fx#556] [General]
Hi!

Am I being stupid or are the libraries missing from maven central?

Readme says:

<dependencies>
  <dependency>
    <groupId>io.fair-acc</groupId>
    <artifactId>chartfx</artifactId>
    <version>11.3.0</version>
  </dependency>
</dependencies>

However: 

https://search.maven.org/search?q=g:io.fair-acc%20AND%20a:chartfx

Yield's zero results.

Many thanks,

Tim

**Comment by RalphSteinhagen** [comment]
@bringmethebacon the 11.3.0 version hasn't been released yet. There is still some work in progress (see #527 and notable #527 + #529 for details). We have been busy with other things and got a bit delayed/side-tracked. but hope to prepare a release in the coming month.

**Reply by bringmethebacon** [reply]
Hey Ralph, great - thanks for letting me know :) 

Tim.
